src/layouts/Sidebar.jsx

```diff
@@ -148,17 +149,21 @@ const Sidebar = ({ menuList, activeMenuItem }) => {
         )}
         <div className="nav-links">
           <ul>
-            {!isDashboardPage && (
+            {/* {!isDashboardPage && (
               <li className={`main-link`}>
                 <a href={"/dashboard"} className="navigation flex">
                   <img src="/icons/backIcon.svg" alt={"Back to Dashboard"} />
                   <span>{"Back to Dashboard"}</span>
                 </a>
               </li>
-            )}
+            )} */}
+
             {menuList &&
               menuList.map((menu, index) => {
-                return (
+                const isCheck = user.role !== "Student";
+                const Render = isCheck || menu.label !== "Applications";
+
```

Comment:
instead of handling it here

  const menuList = [
    { label: "Applications", id: "applications" },
    { label: "Task", id: "task" },
    { label: "Assessment", id: "assessment" },
  ]; change it here by addition condition

Reply:
Changed as you said